app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -188,11 +192,13 @@ public void onResponse(Call<NewsApiResponse> call, Response<NewsApiResponse> res
                 }
                 newsStorage.insertUnique(getNewsItemListByResponse(response, category));
                 updateData();
+                progressBar.setVisibility(ProgressBar.INVISIBLE);
```

Comment:
Can't we use `GONE` instead of `INVISIBLE` as it will remove it from the view hierarchy then.